promscale/quick-start.md

```diff
@@ -0,0 +1,95 @@
+and log in as `admin` with the password `admin`.
+
+### Data sources
+The data sources that are configured for Promscale are:
+* Promscale-PromQL: PromQL query endpoint
+* Promscale-Tracing: Jaeger query endpoint
+* Promscale-SQL: SQL query endpoint for both metrics and traces
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/promscale-grafana-datasource-qsg.png" alt="Promscale data sources"/>
+
+### Application performance monitoring dashboards
+You can use Promscale for application/service performance monitoring using trace data. 
+The Docker Compose tool includes some pre-configured dashboards 
+for you to explore.
+
+#### Overview of services
+View all the services with service name, number of requests served per second,
+average and p90 request duration and error rate. Click the service name
+to view the service level metrics.
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/apm-services-overview-dashboard.png" alt="A dashboard representing the overview of services"/>
+
+#### Slowest requests
+View the top 50 slowest requests across all services with their corresponding traceID.
+Click the traceID to access the gantt chart view of the corresponding trace.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/apm-slowest-traces-dashboard.png" alt="A dashboard representing the slowest requests across the services"/>
+
+#### Rate Error Duration metrics
+View the Rate, Error, Duration (RED) metrics specific to a service to understand
+the number of requests per second served by a service, the number of failed
+requests per second, the amount of time it takes to process requests.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/apm-red-metrics-dashboard.png" alt="A dashboard representing the RED metrics graphs specific to a service"/>
+
+#### Upstream dependencies
+View the upstream dependencies for the selected service and operation. In other words, view a map of
+all the services and operations that were called across all requests before the selected service and 
+operation was called. This helps identify unexpected behaviors like calls between services and operations
+that were not part of the original design. It also accelerates the investigation of the root cause of changes
+in the RED metrics for the selected service and operation. For example, if there is a significant increase in
+the number of requests a particular operation called from many different services is receiving, you can
+track down which service and operation is at the origin of that increase.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/apm-upstream-dependency-dashboard.png" alt="A dashboard representing the upstream dependencies specific to a service"/>
```

Comment:
This image is returning access denied. Have you made it public in s3?

Reply:
Thank you!!! Just enabled read access.